CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [Unreleased]
+
+### Fixed
+- Improved detection of changes in imported data.
```

Comment:
In imported stops?

Reply:
sure, but other models like `TariffZone` and `TopographicPlace` will also benefit from the improved comparison method.